### DISCUSSION TITLE
Domain

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [priyankarpal/projectshut#2412] [General]
## Does Projectshut have a domain yet?

**[porkbun](https://porkbun.com/)** is a really good option to get a custom domain for a great price.

#### Here is an example:

![domain](https://github.com/priyankarpal/projectshut/assets/25883220/f53299d1-9433-423c-bf73-0aaac2a07a79)

**Comment by priyankarpal** [comment]
I don't need it right now. Thank you!
